### ISSUE TITLE
Linkify "getter"/"setter" in the IDL

### ISSUE THREAD

**Issue by tabatkins** [tabatkins/bikeshed#670]
Like I do with some of the other "behavioral" keywords.

**Comment by domenic** [comment]
Slightly related, i.e. HTML currently does this weirdly so don't copy them: https://github.com/whatwg/html/issues/922

**Comment by travisleithead** [comment]
Please have it produce links compatible with `dfn`s of the following (for consistency with how the params are generated today). For example, a getter definition might be:
`<dfn method for="SomeInterface" lt="__getter__()|__getter__(index)">when indexing object...</dfn>`

**Comment by tabatkins** [comment]
Yeah, giving it a pretend name of `__getter__`/`__setter__` is probably fine.
